packages/compose-refs/src/compose-refs/compose-refs.ts

```diff
@@ -0,0 +1,35 @@
+// From Radix Primitives: https://github.com/radix-ui/primitives/blob/main/packages/react/compose-refs/src/composeRefs.tsx
```

Comment:
You can see in the link in that file that the package is for internal use in radix only, so it would be unsafe to install this directly as it can contain breaking changes on any update. 

For each package the only file at the root of `src` is `index.ts`, anything else is contained in a folder. So being consistent with that logic here.